src/lib/enr/enr.spec.ts

```diff
@@ -8,30 +9,46 @@ import { bytesToHex, hexToBytes } from "../utils";
 import { ERR_INVALID_ID } from "./constants";
 import { ENR } from "./enr";
 import { createKeypairFromPeerId } from "./keypair";
+import { Waku2 } from "./waku2_codec";
 
 import { v4 } from "./index";
 
+use(chaibytes);
+
 describe("ENR", function () {
   describe("Txt codec", () => {
-    it("should encodeTxt and decodeTxt", async () => {
+    it.only("should encodeTxt and decodeTxt", async () => {
       const peerId = await PeerId.create({ keyType: "secp256k1" });
       const enr = ENR.createFromPeerId(peerId);
       const keypair = createKeypairFromPeerId(peerId);
       enr.setLocationMultiaddr(new Multiaddr("/ip4/18.223.219.100/udp/9000"));
       enr.multiaddrs = [
```

Comment:
what's also strange is that if you comment out the `multiaddrs` completely, it still throws the `Error: data long segment too short` error!

Reply:
This has been resolved as it was due to a bug with the base64 codec, right?

Reply:
Yes!